src/hotspot/share/utilities/debug.hpp

```diff
@@ -163,19 +163,21 @@ extern bool Debugging;
 
 // error reporting helper functions
 void report_vm_error(const char* file, int line, const char* error_msg);
+ATTRIBUTE_PRINTF(4, 5)
 void report_vm_error(const char* file, int line, const char* error_msg,
-                     const char* detail_fmt, ...) ATTRIBUTE_PRINTF(4, 5);
+                     const char* detail_fmt, ...);
 void report_vm_status_error(const char* file, int line, const char* error_msg,
                             int status, const char* detail);
-void report_fatal(VMErrorType error_type, const char* file, int line, const char* detail_fmt, ...) ATTRIBUTE_PRINTF(4, 5);
+ATTRIBUTE_PRINTF(4, 5) void report_fatal(VMErrorType error_type, const char* file, int line, const char* detail_fmt, ...);
+ATTRIBUTE_PRINTF(5, 6)
 void report_vm_out_of_memory(const char* file, int line, size_t size, VMErrorType vm_err_type,
-                             const char* detail_fmt, ...) ATTRIBUTE_PRINTF(5, 6);
```

Comment:
Why were the ATTRIBUTE_PRINTFs removed?

Reply:
The ATTRIBUTE_PRINTF macros are still there, just moved in front of the methods

Reply:
Wow I'm really having eyesight problems today! Sorry about that.

Reply:
Haha, no worries